### DISCUSSION TITLE
Can config migration be used for shareable config preset repositories?

### DISCUSSION THREAD

**Discussion by marcphilipp** [renovatebot/renovate#31351] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I'm about to extract a [config preset](https://docs.renovatebot.com/config-presets/#github) into separate `renovate-config` repository so that it can be reused. I'm wondering whether [config migration](https://docs.renovatebot.com/configuration-options/#configmigration) works against the `default.json` file in the `renovate-config` repository, given Renovate is active on the `renovate-config` repository.

**Comment by viceice** [comment]
no, not yet supported

**Reply by marcphilipp** [reply]
Thanks for the quick reply! Any plans to add that?

**Reply by viceice** [reply]
yes

**Reply by rarkins** [reply]
Let's create or link to an issue then close this discussion

**Reply by marcphilipp** [reply]
@viceice Is there an existing issue?

**Reply by viceice** [reply]
maybe 🤷‍♂️

**Reply by marcphilipp** [reply]
The closest I found was #22525. @rarkins Please correct me if I'm misinterpreting that issue.
